public/app/ui/components/collections/library/thumbnails-result.html

```diff
@@ -1,5 +1,5 @@
 <a class="thumbnail-result thumbnail-result-library" data-bind="attr: { href: detailUrlFor(result) }">
     <div class="pull-right label label-default" data-bind="component: displayFor('type', result)"></div>
     <div data-bind="component: displayFor('MediaSmall', result)"></div>
-    <div class="caption" data-bind="component: displayForLabelField(result)"></div>
+    <div class="caption" data-bind="component: displayFor('Title', result)"></div>
```

Comment:
I thought we'd already changed these, but apparently we only did it for the `list` view and not `thumbnails` view.  So this is really an unrelated change but needs to be done so here it is.